server/lib/util.py

```diff
@@ -114,6 +114,13 @@ def get_disaster_event_config():
   return get_subject_page_config(filepath)
 
 
+# Returns disaster event config loaded as SubjectPageConfig protos
+def get_disaster_sustainability_config():
+  filepath = os.path.join(get_repo_root(), "config", "disaster_dashboard",
```

Comment:
since sustainability pages and event pages and disaster dashboards are all technically independent "apps", maybe we can rename this config folder to "subject_page"

Reply:
good point - done. they are only related now by the fact that they will link to each other.